src/main/resources/migration/common/V13__audit_affected_components.sql

```diff
@@ -0,0 +1,21 @@
+drop table audit_event;
+
+create table audit_event
+(
+    id                        bigserial primary key not null,
+    csip_record_id            bigserial             not null,
+    action                    varchar(40)           not null,
+    description               text                  not null,
+    actioned_at               timestamp             not null,
+    actioned_by               varchar(32)           not null,
+    actioned_by_captured_name varchar(255)          not null,
+    source                    varchar(12)           not null,
+    active_case_load_id       varchar(6),
+    affected_components        varchar[]             not null,
+    foreign key (csip_record_id) references csip_record (record_id)
```

Comment:
Postgres doesn't add foreign key indexes by default so I think you'll need to add a create index statement for this one.